<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/res.qrc">
    <normaloff>:/image/Setting.svg</normaloff>:/image/Setting.svg</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listWidgetSetting">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidgetSetting">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollAreaGeneral">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>364</width>
            <height>385</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>9</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBoxClickCloseToTray">
             <property name="text">
              <string>关闭窗体时最小化到托盘</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxRunWithWindows">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;由于此功能需要写入注册表，所以当您想删除此软件时，请先取消勾选此项，以免有残留信息留在您的电脑内。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>开机启动程序</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>9</number>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="QComboBox" name="comboBoxSnapType">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>9</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>截图热键</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QKeySequenceEdit" name="keySequenceEditHotKey">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButtonDeleteHotKey">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>30</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="palette">
                    <palette>
                     <active>
                      <colorrole role="ButtonText">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>0</green>
                         <blue>127</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="PlaceholderText">
                       <brush brushstyle="SolidPattern">
                        <color alpha="128">
                         <red>0</red>
                         <green>0</green>
                         <blue>0</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </active>
                     <inactive>
                      <colorrole role="ButtonText">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>255</red>
                         <green>0</green>
                         <blue>127</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="PlaceholderText">
                       <brush brushstyle="SolidPattern">
                        <color alpha="128">
                         <red>0</red>
                         <green>0</green>
                         <blue>0</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </inactive>
                     <disabled>
                      <colorrole role="ButtonText">
                       <brush brushstyle="SolidPattern">
                        <color alpha="255">
                         <red>120</red>
                         <green>120</green>
                         <blue>120</blue>
                        </color>
                       </brush>
                      </colorrole>
                      <colorrole role="PlaceholderText">
                       <brush brushstyle="SolidPattern">
                        <color alpha="128">
                         <red>0</red>
                         <green>0</green>
                         <blue>0</blue>
                        </color>
                       </brush>
                      </colorrole>
                     </disabled>
                    </palette>
                   </property>
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>X</string>
                   </property>
                   <property name="autoDefault">
                    <bool>false</bool>
                   </property>
                   <property name="flat">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBoxIsAutoSave">
                <property name="text">
                 <string>截图自动保存到下列目录</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>9</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>路径</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEditAutoSavePath">
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>11</pointsize>
                    </font>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="toolButtonAutoSavePath">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>9</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>自动保存图片格式</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBoxAutoSaveExtName">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="font">
                    <font>
                     <family>Arial</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="editable">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelWaitTime">
                <property name="text">
                 <string>截图前等待时间</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="horizontalSliderWaitTime">
                <property name="minimum">
                 <number>0</number>
                </property>
                <property name="maximum">
                 <number>20</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="tracking">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="invertedControls">
                 <bool>false</bool>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
                <property name="tickInterval">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>9</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>主题文件</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditQssPath">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonQssPath">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSnap">
      <attribute name="title">
       <string>Snap</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollAreaGeneral_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>364</width>
            <height>385</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>9</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QCheckBox" name="checkBoxIncludeCursor">
             <property name="text">
              <string>截取光标</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxHotKeyNoWait">
             <property name="text">
              <string>热键截图时不等待时间</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxCopyToClipBoardAfterSnap">
             <property name="text">
              <string>截图后立即复制到剪贴板</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxNoBorder">
             <property name="text">
              <string>不截取窗体边框</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxPlaySound">
             <property name="text">
              <string>截图时播放提示音</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>9</number>
              </property>
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>截图方式</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBoxSnapMethod">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollAreaGeneral_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>364</width>
            <height>385</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>9</number>
           </property>
           <property name="leftMargin">
            <number>9</number>
           </property>
           <property name="topMargin">
            <number>9</number>
           </property>
           <property name="rightMargin">
            <number>9</number>
           </property>
           <property name="bottomMargin">
            <number>9</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>打开配置文件...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
